### PR TITLE
Update PowerShell 7.2 to SDK 7.2.1 

### PR DESCRIPTION
**Pull request by Francisco-Gamino**
### Issue describing the changes in this PR

This PR contains the following changes:
* Update `PowerShell 7.2` to SDK `7.2.1`
* Update `.NET` SDK to `6.0.101`

### Pull request checklist

* [x] My changes **do not** require documentation changes
  * [ ] Otherwise: Documentation issue linked to PR
* [x] My changes **should not** be added to the release notes for the next release
  * [ ] Otherwise: I've added my notes to `release_notes.md`
* [ ] My changes **do not** need to be backported to a previous version
  * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [ ] I have added all required tests (Unit tests, E2E tests)

### Additional information

Additional PR information

### CODE REVIEW

File: tools/helper.psm1 [hunk 1]
```diff
@@ -17,8 +17,8 @@ $DotnetSDKVersionRequirements = @{
     }
 
     '6.0' = @{
-        MinimalPatch = '100'
-        DefaultPatch = '100'
+        MinimalPatch = '101'
```
Comment (michaelpeng36) [inline tools/helper.psm1:20]: Should the MinimalPatch and DefaultPatch for 3.1 be updated to 416?

**Review by michaelpeng36 — APPROVED** [review]
Added a single comment regarding the appropriate version of the .NET SDK 3.1. Otherwise, LGTM

File: tools/helper.psm1 (hunk 1 above)
Comment (Francisco-Gamino) [reply tools/helper.psm1:20]: Thank you for pointing this out. Yes, we should. Fixed.

**Review by michaelpeng36 — APPROVED** [review]
LGTM

**Pull request merged**